diff -ruN pendulum-3.0.0/pyproject.toml pendulum-3.0.0-patch/pyproject.toml
--- pendulum-3.0.0/pyproject.toml	2023-12-16 14:06:01.000000000 -0700
+++ pendulum-3.0.0-patch/pyproject.toml	2023-12-17 18:24:01.209934743 -0700
@@ -20,7 +20,6 @@
 
 dependencies = [
     "python-dateutil>=2.6",
-    "tzdata>=2020.1",
     'backports.zoneinfo>=0.2.1; python_version < "3.9"',
     'time-machine>=2.6.0; implementation_name != "pypy"',
     'importlib-resources>=5.9.0; python_version < "3.9"'
